packages/ra-ui-materialui/src/input/AutocompleteArrayInput.stories.tsx

```diff
@@ -399,20 +399,8 @@ const BookEditWithReferenceAndCreationSupport = () => (
         }}
     >
         <SimpleForm>
-            <ReferenceArrayInput reference="authors" source="author" fullWidth>
-                <AutocompleteArrayInput
-                    create={<CreateAuthor />}
-                    options={{
```

Comment:
I remove the code syntax error, but I don't know if the Idea was to render custom options here. Seems out of the scope of this story